src/java.desktop/share/classes/java/awt/Desktop.java

```diff
@@ -587,6 +587,8 @@ public void mail() throws IOException {
      * {@code AWTPermission("showWindowWithoutWarningBanner")}
      * permission, or the calling thread is not allowed to create a
      * subprocess
+     * @spec https://www.rfc-editor.org/info/rfc2368
+     *      RFC 2368: The mailto URL scheme
```

Comment:
The javadoc above references ietf.org. This tag references rfc-editor.org. It seems odd to use two different URLs
in the same method specification for the same RFC. I would have thought the ietf one better to use .. although it should be updated to https

Reply:
This work is primarily about adding `@spec` tags, in a semi-automated fashion, using a custom utility that scans comments in the public API, looking for `href`s to "well-known" sites hosting external specifications.  While "mostly automated", I did manually cleanup the references by line-wrapping as appropriate.

It is out of scope at this time to cleanup up the underlying `href`s, although we should look at cleaning up those links later, identifying the latest/preferred URL for the appropriate version of the spec document, which may or may not be the latest overall version of the spec. (For example, the client docs reference HTML 3.2, and should not reference anything more recent.). This cleanup can be done later and needs to be done in conjunction with the relevant teams, and probably not as a semi-automated update.

Reply:
Sorry, but no matter how many times I try, it seems wrong to even for some short amount of time to have @spec point to one place and the javadoc point to another. They should always be consistent.
If it is out of  scope to update the javadoc, then the new spec tag should just match the existing javadoc.

Reply:
For the `rfc-editor.org` that's going to be hard, insofar as there are strong opinions (voiced in an earlier round of review) about using references to the old sites.

Reply:
I see just 18 references matching `href=....ietf.org` so I guess I can manually update those for you.

```
$ grep -r 'href=[^ ]*ietf.org' open/src/java.desktop
open/src/java.desktop/share/classes/sun/awt/image/PNGImageDecoder.java:    See <a href=http://www.ietf.org/rfc/rfc2083.txt>RFC2083</a> for details. */
open/src/java.desktop/share/classes/java/awt/Desktop.java:     * href="http://www.ietf.org/rfc/rfc2368.txt">The mailto URL
open/src/java.desktop/share/classes/java/awt/peer/DesktopPeer.java:     *        <a href="http://www.ietf.org/rfc/rfc2368.txt">RFC2368: The mailto
open/src/java.desktop/share/classes/javax/print/attribute/standard/Compression.java:     * <a href="http://www.ietf.org/rfc/rfc1952.txt">RFC 1952</a>.
open/src/java.desktop/share/classes/javax/print/attribute/standard/MediaSizeName.java: * <a href="http://www.ietf.org/rfc/rfc2911.txt">RFC 2911</a>
open/src/java.desktop/share/classes/javax/print/attribute/standard/Fidelity.java: * <a href="http://www.ietf.org/rfc/rfc2911.txt">RFC 2911</a> Section 15.1 for a
open/src/java.desktop/share/classes/javax/print/attribute/standard/package-info.java: * <a href="http://www.ietf.org/rfc/rfc2911.txt">RFC 2911</a> for more
open/src/java.desktop/share/classes/javax/print/DocFlavor.java: *   <a href="http://www.ietf.org/rfc/rfc2045.txt">RFC 2045</a> and
open/src/java.desktop/share/classes/javax/print/DocFlavor.java: *   <a href="http://www.ietf.org/rfc/rfc2046.txt">RFC 2046</a>) that specifies
open/src/java.desktop/share/classes/javax/print/DocFlavor.java: * <a href="http://www.ietf.org/rfc/rfc2046.txt">RFC 2046</a>, which says the
open/src/java.desktop/share/classes/javax/print/DocFlavor.java:     * <a href="http://www.ietf.org/rfc/rfc2278.txt">
open/src/java.desktop/share/classes/javax/print/package-info.java: * <a href="http://www.ietf.org/rfc/rfc2911.txt">RFC 2911 Internet Printing
open/src/java.desktop/share/classes/javax/print/MimeType.java: * <a href="http://www.ietf.org/rfc/rfc2045.txt">RFC 2045</a> and
open/src/java.desktop/share/classes/javax/print/MimeType.java: * <a href="http://www.ietf.org/rfc/rfc2046.txt">RFC 2046</a>. A MIME type
open/src/java.desktop/share/classes/javax/imageio/plugins/tiff/BaselineTIFFTagSet.java:     * @see <a href="https://tools.ietf.org/html/rfc1951">DEFLATE specification</a>
open/src/java.desktop/share/classes/javax/imageio/plugins/tiff/FaxTIFFTagSet.java: * <a href="https://tools.ietf.org/html/rfc2306.html">TIFF-F</a> (RFC 2036) file.
open/src/java.desktop/share/classes/javax/imageio/metadata/doc-files/tiff_metadata.html:<td><a href="https://tools.ietf.org/html/rfc1950">
open/src/java.desktop/share/classes/javax/imageio/metadata/doc-files/tiff_metadata.html:<a href="https://tools.ietf.org/html/rfc1951">
$ grep -r 'href=[^ ]*ietf.org' open/src/java.desktop | wc -l
      18